index.js

```diff
@@ -15,6 +15,30 @@ module.exports = {
   parser: 'babel-eslint',
   extends: 'airbnb',
   rules: {
-    'valid-jsdoc': ['error']
+    'valid-jsdoc': ['error'],
+
+    // We've decided we don't want this for now as it makes writing JSX more painful.
+    // See https://github.com/airbnb/javascript/issues/1584#issuecomment-335667272
+    "function-paren-newline": 0,
+
+    // False positive on custom propTypes + isRequired. https://github.com/yannickcr/eslint-plugin-react/issues/1389
+    "react/no-typos": 0,
+
+    // See https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/339
+    "jsx-a11y/anchor-is-valid": ["error", {
+      "components": ["Link"],
+      "specialLink": ["to"],
+      "aspects": ["noHref", "invalidHref", "preferButton"],
+    }],
+
+    // See https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/282
+    // We should probably change our code to adhere to the standard here.
```

Comment:
Again can we explain what's going on here and why "we" aren't adhering to the standard here?

Reply:
Actually I think this rule should be inside the monorepo, not here. 

Rather than fixing an incompatibility, it's just used to work around our own code which isn't fully adherent to this rule yet. It's probably not a great idea to make the rule this lax for every consumer as it's a generally good rule to have.